Circle_Sort/circle_sort.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+//Circle sort algorithm can be visualized by drawing concentric circles on an array of integers. 
+//The elements of the array lying on the same circle diametrically opposite to each other are compared and if found in the wrong order they are swapped.
```

Comment:
Divide the sentence, in a single line there should be around 15 -16 words only

Reply:
@somya-kapoor done the requied changes.